pkg/gradle/lockfile/parse.go

```diff
@@ -0,0 +1,39 @@
+package lockfile
+
+import (
+	"bufio"
+	"strings"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+type Parser struct{}
+
+func NewParser() types.Parser {
+	return &Parser{}
+}
+
+func (Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency, error) {
+	var libs []types.Library
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "#") { // skip comments
+			continue
+		}
+
+		// dependency format: group:artifact:version=classPaths
+		dep := strings.Split(line, ":")
+		if len(dep) != 3 { // skip the last line with lists of empty configurations
```

Comment:
is this check enough? i mean that users can have strange (incorrect) lockfiles: https://github.com/tadfisher/gradle2nix/blob/master/gradle/dependency-locks/settings-classpath.lockfile

or it doesn't matter, right?

Reply:
dependencies from this file don't have `=classPaths`. The rest looks default.
I think we can stay this check. If  there are problems with this, we will add more checks.